### PR TITLE
Fix Makefile for Great Justice

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by MHBauer** [comment]
What's the deal here?

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,8 @@ TEST_DIRS     = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
 SRC_PKGS      = $(addprefix $(SC_PKG)/,$(SRC_DIRS))
 GO_VERSION    = 1.7.3
 GO_BUILD      = go build
-export GOPATH = $(shell cd ../../../..;pwd):$(PWD)/vendor
+BASE_PATH     = $(ROOT:/src/github.com/kubernetes-incubator/service-catalog/=)
```
Comment (MHBauer) [inline Makefile]: Is there supposed to be an `=` at the end?

**Comment by duglin** [comment]
his dir structure was a bit different, didn't have the normal src/github.com/... path - just "service-catalog" .

File: Makefile (hunk 1 above)
Comment (duglin) [inline Makefile]: yup - its a form of search-n-replace.  src=tgt   in this case tgt is empty

**Comment by arschles** [comment]
:100: on the title of this PR. I missed it the first time around :+1:

File: Makefile (hunk 1 above)
Comment (MHBauer) [inline Makefile]: cool, I learned something.
https://www.gnu.org/software/make/manual/html_node/Substitution-Refs.html

**Comment by bmelville** [comment]
LGTMing and merging

**Pull request merged**